pom.xml

```diff
@@ -652,34 +664,86 @@
         </executions>
       </plugin>
 
+      <!-- Export release notes -->
+      <plugin>
+        <groupId>org.apache.logging.log4j</groupId>
+        <artifactId>log4j-changelog-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <!-- Execution ID must match the one inherited from `logging-parent` (i.e., `export-changelog`) to override it! -->
+            <id>export-changelog</id>
+            <goals>
+              <goal>export</goal>
+            </goals>
+            <phase>pre-site</phase>
+            <inherited>false</inherited>
+            <configuration>
+              <outputDirectory>${project.build.directory}/generated-site/antora/modules/ROOT/pages</outputDirectory>
+              <indexTemplates>
+                <template>
+                  <source>.index.adoc.ftl</source>
+                  <target>release-notes.adoc</target>
+                </template>
+              </indexTemplates>
+              <changelogTemplates>
+                <template>
+                  <source>.release-notes.adoc.ftl</source>
+                  <target>_release-notes/_%v.adoc</target>
+                </template>
+              </changelogTemplates>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <inherited>false</inherited>
+        <executions>
+
+          <!-- Copy `src/site/antora/antora.tmpl.yml` to `target/antora-yml/antora.yml` -->
+          <execution>
+            <id>copy-antora-yml</id>
```

Comment:
No, because `resources:copy-resources` _copies_, but we need to _rename_ here as well. That is, `antora.tmpl.yml` needs to be copied to `antora.yml`. `resources:copy-resources` doesn't support that.